test/utilities.py

```diff
@@ -18,6 +19,62 @@
     )
 
 
+class TempFile:
+    """docstring for TempFile"""
+
+    def __init__(self, name, handle):
+        super(TempFile, self).__init__()
+        self.name = name
+        self.handle = handle
+
+    def read_whole_file(self):
+        while True:
+            if not self.handle.read(10):
+                break
+        return self.handle.read()
+
+    def rwf(self):
+        return self.read_whole_file()
+
+    def close(self):
+        self.handle.close()
+
+
+class TempFileCtx:
+    """docstring for TempFileCtxs"""
```

Comment:
Not very useful :)

Reply:
sorry, its autogenerated by sublime. useful because then you dont have to learn path api in python. and your "jump to definition" hotkey takes you to the same file instead of the python functions.

i had to read the whole file to the EOL and then return file

Reply:
oh you know what this returns nothing, damn...

Reply:
I don't fully understand your reason (I dot use Sublime Text) but ok ^^